### PR TITLE
Add tasks_pending per job metric

### PR DESCRIPTION
**Pull request by dln**
Makes it easy for a job owner to track if a job is in trouble.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -37,9 +38,16 @@ var httpClient = http.Client{
...
-	f        finder
-	errors   prometheus.Counter
-	duration prometheus.Gauge
+	f            finder
+	errors       prometheus.Counter
+	duration     prometheus.Gauge
+	pendingTasks *prometheus.GaugeVec
+}
+
+type PendingTask struct {
```
Comment (tommyulfsparre) [inline main.go:47]: PendingTask  -> pendingTask, don't have to export this

**Comment by tommyulfsparre** [comment]
@dln :star: , just one small nitpick

**Comment by StephanErb** [comment]
+1

**Comment by thinker0** [comment]
+1

**Comment by tommyulfsparre** [comment]
Sorry, forgot about this!

**Pull request merged**